app/components/landing-title/component.js

```diff
@@ -1,7 +1,16 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
+
     session: Ember.inject.service(),
+
+    style: Ember.computed('containerStyle', function() {
+        return this.get('containerStyle');
```

Comment:
This seems super unnecessary

Reply:
i think youre right. might have missed that.